app/webpacker/styles/application.scss

```diff
@@ -4,3 +4,11 @@
 
 @import "status-box";
 @import "summary-list";
+
+.fixed-table {
```

Comment:
This feels out of place here in the main `application.scss`. It should probably live in a `utilities` sub directory